doc/modules/model_evaluation.rst

```diff
@@ -946,15 +946,15 @@ Then the metrics are defined as:
 +---------------+------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------+
 |``average``    | Precision                                                                                                        | Recall                                                                                                           | F\_beta                                                                                                              |
 +===============+==================================================================================================================+==================================================================================================================+======================================================================================================================+
-|``"micro"``    | :math:`P(y, \hat{y})`                                                                                            | :math:`R(y, \hat{y})`                                                                                            | :math:`F_\beta(y, \hat{y})`                                                                                          |
```

Comment:
I am thinking that it would be best to keep `P(y, y_hat)`. This is closer to the `precision_score(y_true, y_pred)` function used to program. We should only have to make sure that the denominator in the definition above is the right one.